azkaban-common/src/main/java/azkaban/storage/LocalStorage.java

```diff
@@ -98,25 +98,40 @@ public String put(final StorageMetadata metadata, final File localFile) {
     try {
       FileUtils.copyFile(localFile, targetFile);
     } catch (final IOException e) {
-      log.error("LocalStorage error in put(): meta: " + metadata);
+      log.error("LocalStorage error in putProject(): meta: " + metadata);
       throw new StorageException(e);
     }
     return getRelativePath(targetFile);
   }
 
-  private String getRelativePath(final File targetFile) {
-    return this.rootDirectory.toURI().relativize(targetFile.toURI()).getPath();
+  @Override
+  public InputStream getDependency(final Dependency dep) throws IOException {
+    throw new UnsupportedOperationException("Dependency fetching is not supported with LocalStorage.");
   }
 
   @Override
-  public boolean delete(final String key) {
-    final File file = getFile(key);
+  public boolean dependencyFetchingEnabled() {
+    return false;
+  }
+
+  @Override
+  public String getDependencyRootPath() {
+    return null;
+  }
```

Comment:
same thing here, please add some comments for why these are not being used(?)

Reply:
It was renamed to `deleteProject`. Git diff is misleading.